### PR TITLE
prepare mineral ui repo to share with {V} comunity

### PR DESCRIPTION
**Pull request by bdebicki**
- prepare circle ci configuration
- add script to run EM
- add short readme file and move original mineral readme file to another file
- update merge cli to 1.3.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,121 +1,27 @@
...
 
-[![build status](https://travis-ci.com/mineral-ui/mineral-ui.svg?branch=master)](https://travis-ci.com/mineral-ui/mineral-ui)
-[![dependency status](https://david-dm.org/mineral-ui/mineral-ui.svg)](https://david-dm.org/mineral-ui/mineral-ui)
-[![managed with Waffle](https://img.shields.io/badge/managed_with-Waffle-72b2e4.svg)](https://waffle.io/mineral-ui/mineral-ui)
-[![Commitizen friendly](https://img.shields.io/badge/Commitizen-friendly-brightgreen.svg?style=flat)](https://github.com/commitizen/cz-cli)
-[![code style Prettier](https://img.shields.io/badge/code_style-Prettier-ff69b4.svg)](https://github.com/prettier/prettier)
+![alt text](https://github.com/uxpin-merge/rmwc-merge/blob/master/img/header.png "RMWC in UXPin")
 
-# [Mineral UI](https://mineral-ui.com/)
+## About RMWC
```
Comment (michalstocki) [inline README.md]:
```suggestion
## About Mineral UI
```

File: README.md [hunk 2]
```diff
@@ -1,121 +1,27 @@
+# Mineral UI + UXPin Merge
+Integration of [Mineral UI](https://mineral-ui.com/) with UXPin Merge Technology.
 
-[![build status](https://travis-ci.com/mineral-ui/mineral-ui.svg?branch=master)](https://travis-ci.com/mineral-ui/mineral-ui)
-[![dependency status](https://david-dm.org/mineral-ui/mineral-ui.svg)](https://david-dm.org/mineral-ui/mineral-ui)
-[![managed with Waffle](https://img.shields.io/badge/managed_with-Waffle-72b2e4.svg)](https://waffle.io/mineral-ui/mineral-ui)
-[![Commitizen friendly](https://img.shields.io/badge/Commitizen-friendly-brightgreen.svg?style=flat)](https://github.com/commitizen/cz-cli)
-[![code style Prettier](https://img.shields.io/badge/code_style-Prettier-ff69b4.svg)](https://github.com/prettier/prettier)
+![alt text](https://github.com/uxpin-merge/rmwc-merge/blob/master/img/header.png "RMWC in UXPin")
```
Comment (michalstocki) [inline README.md]: Better use an image from Mineral UI.

Comment (bdebicki) [reply README.md]: oh yes. we don't add an image for now. i forgot to remove it

File: README.md (hunk 1 above)
Comment (bdebicki) [reply README.md]: 👍

**Pull request merged**